app/interfaceactiontablehandler.py

```diff
@@ -324,9 +324,9 @@ async def get(self, request):
                 previous_jobrole_selected = session['jobRoleShort']
                 search_criteria['jobRoleShort'] = previous_jobrole_selected
 
-            if data.get('filter_external_id'):
-                external_id = data.get('filter_external_id')
-                search_criteria['externalId'] = external_id 
+            if session.get('uniqueEmployeeId'):
+                unique_employee_id = data.get('uniqueEmployeeId')
```

Comment:
This line should probably be merged with the one below, if the `unique_employee_id` variable is never re-used.

Reply:
It will be re-used in a future branch that will fix the persistence of the contents of the  external_id input  (no ticket yet)